### ISSUE TITLE
Suggestion #1

### ISSUE THREAD

**Issue by rcon420** [NotJustin/SkillAutoBalance#41]
could you add cvar to specify how many players needed before player/join leave auto-balance occurs? 

Like for example, I have 10 CTS  and 7 TS I would like to balance if the difference is every 3 players also in my case if Team TS is losing they should get 2 players so it is 9 TS vs  8 CT in my case but if the team is doing good do nothing most likely after 1-2 rounds new player will join.   Otherwise, auto-balance at the current state overall is way too aggressive with all other settings and I think sometimes balance was not even needed with the current system and good players get annoyed. If we are getting too complex for example it could sometimes flip over  2 people and count their total kills together not always move the first person on the top.  TL;DR there is a lot of WORK for the Best Balance Plugin so far you are doing good!

**Comment by NotJustin** [comment]
To clarify:

Currently, 10v10 is ok, 10v9 is ok, 10v8 or less is bad. You want to have it so 10v8 is ok, and 10v7 or less is bad, correct?

It is a good idea. It just means on this line, rather than checking if the difference is `<= 1` , I should check if the difference is less than the convar value you suggest. https://github.com/NotJustin/SkillAutoBalance/blob/master/scripting/sab-checkbalance.sp#L250

**Comment by NotJustin** [comment]
The other issue you describe here about the balance being too aggressive, it depends on many things. I can help you configure it, but I would prefer if you made another issue for it.

**Comment by rcon420** [comment]
> To clarify:
> 
> Currently, 10v10 is ok, 10v9 is ok, 10v8 or less is bad. You want to have it so 10v8 is ok, and 10v7 or less is bad, correct?
> 
> It is a good idea. It just means on this line, rather than checking if the difference is `<= 1` , I should check if the difference is less than the convar value you suggest. https://github.com/NotJustin/SkillAutoBalance/blob/master/scripting/sab-checkbalance.sp#L250

Yes, something like that but in fact checks, if the team is losing a duo fewer players on the team only then do the switch otherwise is unnecessary aggressive on the public casual server since players/leave join all the time and I can't do anything about it. Overall, I see that people like the plugin only everyone complains that is too aggressive and I will make a separate issue.
